lib/grammers-client/src/client/messages.rs

```diff
@@ -751,6 +751,35 @@ impl Client {
         Ok(message_ids.iter().map(|id| map.remove(id)).collect())
     }
 
+    /// Get a message using your ID.
```

Comment:
```suggestion
    /// Fetch a single message by ID.
```

If we could also say, "this is cheaper than `[get_messages]`" (linking to that function), I think that would be useful.

Reply:
oh, i think it would be good